pkgs/development/python-modules/ruamel-yaml-string/default.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "ruamel-yaml-string";
+  version = "0.1.1";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "ruamel.yaml.string";
+    hash = "sha256-enrtzAVdRcAE04t1b1hHTr77EGhR9M5WzlhBVwl4Q1A=";
+  };
+
+  build-system = [ setuptools ];
+
+  pythonImportsCheck = [ "ruamel.yaml.string" ];
```

```suggestion
  dependencies = [ ruamel-yaml ];
```
This fixes the build failure caused by pyproject = true

---

Interesting, didn't know that `pyproject = true` works even for projects without a `pyproject.toml` file.

I've merged your suggestion.